src/components/Home.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable max-len */
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Col, Row } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+// import { getCoins } from '../redux/cryptodata/cryptodata';
+import Filter from './Filter';
```

Comment:
- [ ] Kindly delete all the commented lines of code because they make your work look messy.

Reply:
This change has been implemented well